packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -180,6 +197,61 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
   }
 }
 
+/** check the dotnet sdk installation.
+ * Report diagnostic if dotnet sdk is not installed or its version does not meet prerequisite
+ * @param program The typespec compiler program
+ * @param minVersionRequisite The minimum required version
+ */
+async function validateDotnet(program: Program, minVersionRequisite: string): Promise<boolean> {
```

Comment:
Can we add a unit test of this function?

Reply:
added